src/WebJobs.Script.WebHost/Management/MeshServiceClient.cs

```diff
@@ -6,6 +6,8 @@
 using System.IO;
 using System.Linq;
 using System.Net.Http;
+using System.Net.Http.Formatting;
```

Comment:
Unused ? In Visual studio, right click here and select "Remove and Sort usings" from the context menu.

Reply:
Fixed!